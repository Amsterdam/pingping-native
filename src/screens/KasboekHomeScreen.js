import React from 'react';
import {View, StyleSheet, Dimensions} from 'react-native';
import {Content, Container, Text} from 'native-base';

const styles = StyleSheet.create({});

const KasboekHomeScreen = props => {
  return (
    <Container>
      <Content>
        <Text>KasboekHomeScreen</Text>
      </Content>
    </Container>
  );
};

export default KasboekHomeScreen;
