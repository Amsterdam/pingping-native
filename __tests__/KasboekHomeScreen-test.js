import React from 'react';
import KasboekHomeScreen from '../src/screens/KasboekHomeScreen';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<KasboekHomeScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
