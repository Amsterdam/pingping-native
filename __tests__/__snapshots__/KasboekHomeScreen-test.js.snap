// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "height": 1334,
    }
  }
>
  <View>
    <RCTSafeAreaView
      emulateUnlessSupported={true}
      style={
        Object {
          "backgroundColor": "#0d2036",
        }
      }
    >
      <View
        noShadow={true}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "borderBottomColor": "transparent",
              "borderBottomWidth": 0.5,
              "elevation": 0,
              "flexDirection": "row",
              "height": 64,
              "justifyContent": "center",
              "left": 0,
              "paddingLeft": 6,
              "paddingRight": 10,
              "paddingTop": 18,
              "right": 0,
              "shadowColor": null,
              "shadowOffset": null,
              "shadowOpacity": null,
              "shadowRadius": null,
              "top": 0,
            },
            Object {
              "alignItems": "center",
              "backgroundColor": "#0d2036",
              "flexDirection": "column",
              "justifyContent": "center",
              "margin": 15,
            },
          ]
        }
        transparent={true}
      >
        <Text
          style={
            Array [
              Object {
                "color": "#000",
                "fontFamily": "System",
                "fontSize": 16,
              },
              Object {
                "color": "#fff",
                "fontFamily": "Heavitas",
                "fontSize": 14,
                "textAlign": "left",
              },
            ]
          }
          uppercase={false}
        >
          Zet je goals
        </Text>
      </View>
    </RCTSafeAreaView>
  </View>
  <RCTSafeAreaView
    emulateUnlessSupported={true}
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
      }
    }
  >
    <RCTScrollView
      automaticallyAdjustContentInsets={false}
      contentContainerStyle={
        Array [
          Object {
            "padding": undefined,
          },
          undefined,
        ]
      }
      contentInset={
        Object {
          "bottom": 0,
        }
      }
      enableAutomaticScroll={true}
      enableOnAndroid={false}
      enableResetScrollToCoords={true}
      extraHeight={75}
      extraScrollHeight={0}
      getScrollResponder={[Function]}
      handleOnScroll={[Function]}
      keyboardDismissMode="interactive"
      keyboardOpeningTime={250}
      keyboardShouldPersistTaps="handled"
      keyboardSpace={0}
      onScroll={[Function]}
      resetKeyboardSpace={[Function]}
      resetScrollToCoords={
        Object {
          "x": 0,
          "y": 0,
        }
      }
      scrollEventThrottle={1}
      scrollForExtraHeightOnAndroid={[Function]}
      scrollIntoView={[Function]}
      scrollToEnd={[Function]}
      scrollToFocusedInput={[Function]}
      scrollToPosition={[Function]}
      showsVerticalScrollIndicator={true}
      style={
        Object {
          "backgroundColor": "transparent",
          "flex": 1,
        }
      }
      update={[Function]}
      viewIsInsideTabBar={false}
    >
      <View>
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#0d2036",
              "padding": 10,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#fff",
                "borderRadius": 20,
                "marginBottom": 30,
                "padding": 20,
                "width": 700,
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#000",
                    "fontFamily": "System",
                    "fontSize": 16,
                  },
                  Object {
                    "fontFamily": "Heavitas",
                    "fontSize": 30,
                    "textAlign": "left",
                  },
                ]
              }
              uppercase={false}
            >
              0 Goals
            </Text>
            <Text
              style={
                Array [
                  Object {
                    "color": "#000",
                    "fontFamily": "System",
                    "fontSize": 16,
                  },
                  Object {
                    "fontFamily": "Raleway",
                    "fontSize": 15,
                    "lineHeight": 25,
                    "textAlign": "left",
                  },
                ]
              }
              uppercase={false}
            >
              Je hebt nog geen goals opgesteld
            </Text>
            <View
              style={
                Object {
                  "alignSelf": "stretch",
                  "flex": 1,
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "marginTop": 20,
                }
              }
            >
              <View
                style={
                  Object {
                    "alignItems": "center",
                  }
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#000",
                        "fontFamily": "System",
                        "fontSize": 16,
                      },
                      Object {
                        "fontFamily": "Raleway",
                        "fontSize": 15,
                        "lineHeight": 25,
                        "textAlign": "left",
                      },
                    ]
                  }
                  uppercase={false}
                >
                  Nieuwe goals
                </Text>
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#000",
                        "fontFamily": "System",
                        "fontSize": 16,
                      },
                      Object {
                        "fontFamily": "Heavitas",
                        "fontSize": 30,
                        "textAlign": "left",
                      },
                    ]
                  }
                  uppercase={false}
                >
                  0
                </Text>
              </View>
              <View
                style={
                  Object {
                    "alignItems": "center",
                  }
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#000",
                        "fontFamily": "System",
                        "fontSize": 16,
                      },
                      Object {
                        "fontFamily": "Raleway",
                        "fontSize": 15,
                        "lineHeight": 25,
                        "textAlign": "left",
                      },
                    ]
                  }
                  uppercase={false}
                >
                  Bijna afgerond
                </Text>
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#000",
                        "fontFamily": "System",
                        "fontSize": 16,
                      },
                      Object {
                        "fontFamily": "Heavitas",
                        "fontSize": 30,
                        "textAlign": "left",
                      },
                    ]
                  }
                  uppercase={false}
                >
                  0
                </Text>
              </View>
              <View
                style={
                  Object {
                    "alignItems": "center",
                  }
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#000",
                        "fontFamily": "System",
                        "fontSize": 16,
                      },
                      Object {
                        "fontFamily": "Raleway",
                        "fontSize": 15,
                        "lineHeight": 25,
                        "textAlign": "left",
                      },
                    ]
                  }
                  uppercase={false}
                >
                  afgerond
                </Text>
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#000",
                        "fontFamily": "System",
                        "fontSize": 16,
                      },
                      Object {
                        "fontFamily": "Heavitas",
                        "fontSize": 30,
                        "textAlign": "left",
                      },
                    ]
                  }
                  uppercase={false}
                >
                  0
                </Text>
              </View>
            </View>
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "bottom": 0,
                "justifyContent": "center",
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 225,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "elevation": 5,
                  "height": 100,
                  "justifyContent": "center",
                  "shadowColor": "#000",
                  "shadowOffset": Object {
                    "height": 2,
                    "width": 0,
                  },
                  "shadowOpacity": 0.25,
                  "shadowRadius": 3.84,
                  "width": 200,
                }
              }
            >
              <View
                accessible={true}
                focusable={true}
                forwardedRef={[Function]}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "alignSelf": "stretch",
                    "backgroundColor": "#fb9f00",
                    "borderBottomWidth": null,
                    "borderColor": false,
                    "borderLeftWidth": null,
                    "borderRadius": 57,
                    "borderRightWidth": null,
                    "borderTopWidth": null,
                    "borderWidth": undefined,
                    "elevation": 5,
                    "flexDirection": "row",
                    "height": 45,
                    "justifyContent": "center",
                    "marginBottom": 5,
                    "opacity": 1,
                    "paddingBottom": 6,
                    "paddingTop": 6,
                    "shadowColor": "#000",
                    "shadowOffset": Object {
                      "height": 2,
                      "width": 0,
                    },
                    "shadowOpacity": 0.25,
                    "shadowRadius": 3.84,
                  }
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "backgroundColor": "transparent",
                        "color": "#fff",
                        "fontFamily": "System",
                        "fontSize": 16.5,
                        "marginLeft": 0,
                        "marginRight": 0,
                        "paddingLeft": 16,
                        "paddingRight": 16,
                      },
                      Object {
                        "color": "#fff",
                        "fontFamily": "Heavitas",
                        "fontSize": 14,
                      },
                    ]
                  }
                  uppercase={false}
                >
                  zet nieuwe goal
                </Text>
              </View>
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "alignSelf": "stretch",
              "justifyContent": "center",
              "marginTop": 40,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#fff",
                "elevation": 5,
                "flex": 1,
                "flexDirection": "row",
                "margin": 20,
                "marginBottom": 20,
                "padding": 20,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 2,
                  "width": 0,
                },
                "shadowOpacity": 0.25,
                "shadowRadius": 3.84,
              }
            }
          >
            <Text
              allowFontScaling={false}
              style={
                Array [
                  Object {
                    "color": "#000",
                    "fontSize": 12,
                  },
                  Array [
                    Object {
                      "color": "#000",
                      "fontSize": 30,
                    },
                    Object {
                      "color": "#000",
                      "fontSize": 30,
                    },
                    Object {
                      "fontSize": 40,
                      "marginBottom": 5,
                      "marginRight": 20,
                    },
                  ],
                  Object {
                    "fontFamily": "FontAwesome5Free-Solid",
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  },
                  Object {
                    "fontWeight": "900",
                  },
                ]
              }
              type="FontAwesome5"
            >
              
            </Text>
            <Text
              style={
                Array [
                  Object {
                    "color": "#000",
                    "fontFamily": "System",
                    "fontSize": 16,
                  },
                  Object {
                    "fontFamily": "Heavitas",
                    "fontSize": 15,
                    "textAlign": "left",
                  },
                ]
              }
              uppercase={false}
            >
              € 100 verjaardagsfeestje
            </Text>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </RCTSafeAreaView>
</View>
`;
